### ISSUE TITLE
Integrating with tekton builds

### ISSUE THREAD

**Issue by sfxworks** [konveyor/pelorus#261]
Not sure if this is similar to https://github.com/konveyor/pelorus/issues/224, though I would like to have pelorus integrated with TektonCD image builds as Jenkins with OpenShift https://docs.openshift.com/container-platform/4.6/builds/build-strategies.html#builds-strategy-pipeline-build_build-strategies

**Comment by malacourse** [comment]
I have started looking at this feature and have a version in my fork: https://github.com/redhat-mal/pelorus/blob/tekton-dev/exporters/committime/collector_base.py

This solved the use case where a binary build is done in Tekton based on a Pipeline Git Resource.  I had to create a custom builder that is based on Maven but also includes Git so the resulting build has the Git info as far as commit.  I plan on adding this to 1.4 when time permits.
